chapters/config.Rmd

```diff
@@ -233,7 +233,7 @@ mpl.get_configdir()
 /Users/amira/.matplotlib
 ```
 
-Once we've created the new sub-directory,
+Once we've created the new sub-directory:
```

Comment:
I think it's pretty uncontroversial to say that we should use `:` when the code block comes at the end of a sentence, but doesn't it make more sense to use `,` when the code block is introduced mid-sentence, like it is in this example?

Reply:
This. Is. PYTHON!!

Reply:
🤣